packages/babel-core/test/api.js

```diff
@@ -118,26 +118,31 @@ describe("api", function() {
   });
 
   it("transformFile", function(done) {
-    babel.transformFile(
-      __dirname + "/fixtures/api/file.js",
-      {
-        babelrc: false,
-      },
-      function(err, res) {
-        if (err) return done(err);
-        assert.equal(res.code, "foo();");
-        done();
-      },
-    );
+    const options = {
+      babelrc: false,
+    };
+    babel.transformFile(__dirname + "/fixtures/api/file.js", options, function(
+      err,
+      res,
+    ) {
+      if (err) return done(err);
+      assert.equal(res.code, "foo();");
+      // keep user options untouched
+      assert.equal(Object.keys(options).length, 1);
```

Comment:
This isn't enough, since if a property value is modified it won't be caught.
```js
assert.deepEqual(options, { babelrc: false });
```

Reply:
Yes you are right, I missed that.